pkg/apis/kops/dockerconfig.go

```diff
@@ -60,6 +62,8 @@ type DockerConfig struct {
 	MTU *int32 `json:"mtu,omitempty" flag:"mtu"`
 	// RegistryMirrors is a referred list of docker registry mirror
 	RegistryMirrors []string `json:"registryMirrors,omitempty" flag:"registry-mirror,repeat"`
+	// Runtimes registers an additional OCI compatible runtime (default [])
+	Runtimes []string `json:"addRuntime,omitempty" flag:"add-runtime,repeat"`
```

Comment:
Shouldn't this match also?
```suggestion
	Runtimes []string `json:"runtimes,omitempty" flag:"add-runtime,repeat"`
```

Reply:
This is very embarrassing haha! Eitherways will rework it!

Reply:
No worries. It happens all the time with these names.

Reply:
:)

Reply:
pushed final changes. hopefully no misses this time.

Reply:
Looks quite good now. Thanks!